web/components/PetPostModal/ImageSlider.tsx

```diff
@@ -49,20 +50,43 @@ function ImageSlider(props: { attachments: Array<string> }) {
         );
       }}
     >
-      {props.attachments?.map((slide) => {
-        return (
-          <Image
-            key={slide}
-            src={slide}
-            borderRadius="15px"
-            objectFit="cover"
-            width="full"
-            height="full"
-            maxHeight={["50vh", "75vh"]}
-            alt=""
-          />
-        );
-      })}
+      {/* TODO: Account for Videos */}
+      {props.attachments.length > 0
+        ? props.attachments?.map((slide) => {
+            return (
+              <Flex
+                justifyContent={"center"}
+                alignItems={"center"}
+                bgColor={"#C6E3F9"}
+                borderRadius="15px"
+                minHeight={["40vh", "75vh"]}
+                key={slide}
+              >
+                <Image
+                  src={slide}
+                  objectFit="cover"
+                  verticalAlign={"true"}
+                  align="center"
+                  maxHeight={["40vh", "75vh"]}
+                  borderRadius="15px"
+                  alt=""
+                />
+              </Flex>
+            );
+          })
+        : [
+            <Flex
+              key={"default"}
+              direction={"column"}
+              justifyContent={"center"}
+              alignItems={"center"}
+              borderRadius="15px"
+              maxHeight={["40vh", "70vh"]}
+              paddingBottom={10}
+            >
+              <DefaultDog fill="#C6E3F9" height={"100%"} width={"100%"} />
```

Comment:
Keep this dog icon white to correspond to the Figma, you can also look at `FeedPostCard` to see how we render it there